jobCreationProcess.js

```diff
@@ -0,0 +1,60 @@
+import { describe, it, expect } from 'vitest';
+
+describe("Successful job creation", () => {
+
+    //Sucessful job creation
+    it("Should return 200 OK with job details (ID, title, description, created date)", async () => {
+        const res = await request(app)
+            .get("/api/job")
+            .set(...(await gt()))
+            .send();
+
+        expect(res.status).toBe(200);
+        expect(res.body).toHaveProprty("jobId");
+        expect(res.body.title).toBe("title");
+        expect(res.description).toBe("description");
+        expect(res.body).toHaveProperty("createdDate");
+    });
+    
+    //Missing Required Fields
+    it("Should return 400 bad request with relevant error message", async () => {
+        const res = await request(app)
+            .get("/api/job")
+            .send({});
```

Comment:
This request will always return 401, since we arent passing an auth header!

```suggestion
        const res = await request(app)
            .get("/api/job")
            .set(...(await gt()))
            .send({});
```